server/routes/teamAPI.js

```diff
@@ -0,0 +1,60 @@
+const express = require("express");
+const db = require("../db/models");
+const auth = require("../middleware/auth");
+
+const router = express.Router();
+
+// CREATE team
+router.post("/api/team/create", auth, (req, res) => {
+  db.Team.create(req.body)
+    .then((response) => res.send(response))
+    .catch((error) => {
+      console.log(error);
+      res.status(500).send(error);
+    });
+});
+
+// ADD Team member
+router.get("/api/team/add", auth, (req, res) => {
+  db.Team.updateOne({ _id: req.body.teamId }, { $push: req.body.member })
```

Comment:
Shouldn't this be { $push: { members: req.body.member } })?

Reply:
I'm thinking the same unless team members are somehow pushed directly onto the array.

Reply:
Yes, it should!